sound/soc/sof/intel/hda.h

```diff
@@ -338,6 +338,8 @@
 #define HDA_ADSP_FW_STATUS_SKL		HDA_ADSP_SRAM0_BASE_SKL
 #define HDA_ADSP_ERROR_CODE_SKL		(HDA_ADSP_FW_STATUS_SKL + 0x4)
 
+#define HDMI_CODEC(x) ((x) & BIT(2))
```

Comment:
HDA_IDISP_CODEC
let's make the macros consistent in terms of prefix

Reply:
yes, I will refine it. Thanks!